lib/utils/resource-links.js

```diff
@@ -0,0 +1,60 @@
+function isResourceLink (field) {
+  return field && field.sys && field.sys.type === 'ResourceLink'
+}
+
+function getResourceLinkFields (data) {
+  const items = [].concat(...(data.items || []), ...Object.values(data.includes || {}).flat())
+  const fields = items.flatMap(({ fields }) => Object.values(fields).flat())
+  return fields.filter(isResourceLink)
```

Comment:
This code breaks if the `locale=*` parameter is set as it assumes the field response does not include the localisation map

Reply:
Good point, I'll look into this